config/gocdb_schema.xml

```diff
@@ -163,6 +163,12 @@
 	    <length>255</length>
 	    <regex>/^(([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6}[,;]?)+$/</regex>
 	</field>
+  <field>
```

Comment:
This block is appears less indented than it's neighbouring `field` blocks :cry:

Reply:
This file has a mix of tabs and space indentation. The lines I have touched should now be in line with practice elsewhere in GOCDB.